### ISSUE TITLE
y everyone 4-6? do i need to change some parameter?

### ISSUE THREAD

**Issue by kelisima001** [spmallick/learnopencv#649]
![image](https://user-images.githubusercontent.com/35684777/157597677-7391c995-6fb9-4b14-8f2c-86cdd47eabec.png)

**Comment by brmarkus** [comment]
Excuse me, can you rephrase your question, please?
Can you describe what you see, what you have expected instead?
Which application do you use, how have you called it, which parameter have you used, have you changed something, do you use the same NN-model underneath?

**Comment by kelisima001** [comment]
i used python3.8
opencv 4.5.3.56
i downloaded age_net.caffemodel and  gender_net.caffemodel and then putted under AgeGender
else did not change anything
then i used python AgeGender.py --input "C:\\Users\\xxx\\Pictures\\2110021F20422A.jpg"
show this

**Comment by kelisima001** [comment]
i git clone the project and cd to AgeGender
in windows

**Comment by brmarkus** [comment]
Cannot comment on the age-network's precision or what material was used for training, whether it used "representative" images, including Asian faces of women in different ages...
Can you try other images showing different men, woman of different age, please?

Random examples:
https://upload.wikimedia.org/wikipedia/commons/2/2d/Angela_Merkel_Juli_2010_-_3zu4.jpg
https://upload.wikimedia.org/wikipedia/commons/a/af/Claudia_Schiffer_C%C3%A9sars_1993.jpg

**Comment by chinlee1523** [comment]
<img width="540" alt="image" src="https://user-images.githubusercontent.com/3362244/159681341-365ab8a5-dedf-4222-ad7f-26bb488458a6.png">
I found the same thing

**Comment by brmarkus** [comment]
You might contact "https://github.com/GilLevi/AgeGenderDeepLearning" and check "https://talhassner.github.io/home/publication/2015_CVPR" for which material was used to train the model. It could mean that the selected data is not representative enough.
